modules/structs/repo.go

```diff
@@ -122,6 +123,8 @@ type CreateRepoOption struct {
 	// TrustModel of the repository
 	// enum: default,collaborator,committer,collaboratorcommitter
 	TrustModel string `json:"trust_model"`
+	// MirrorInterval time when creating a mirror (used with mirrors)
+	MirrorInterval string `json:"mirror_interval"`
```

Comment:
this should not go into here ... since on normal repo creation no mirror will be ever created

onyl throu migration api

Reply:
Removed this from here and the API where I added it improperly.